public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/CreateNew/CreateNew.js

```diff
@@ -0,0 +1,474 @@
+      <EuiText size="xs">
+        <p>
+          Create and configure an anomaly detector to automatically detect anomalies in your data and 
+          to view real-time results on the visualization. {' '}
+          <a
+            href="https://opensearch.org/docs/latest/observing-your-data/ad/index/"
+            target="_blank"
+          >
+            Learn more <EuiIcon type="popout" />
+          </a>
+        </p>
+      </EuiText>
+      <EuiSpacer size="m" />
+      <div className="create-new__title-and-toggle">
+        <EuiTitle size="xxs">
+          <h4>
+            <EuiIcon type="visLine" className="create-new__title-icon" />
+            {title}
+          </h4>
+        </EuiTitle>
+        <EuiSwitch
+          label="Show visualization"
+          checked={isShowVis}
+          onChange={() => setIsShowVis(!isShowVis)}
+        />
+      </div>
+      <div className={`create-new__vis ${!isShowVis && 'create-new__vis--hidden'}`}>
+        <EuiSpacer size="s" />
+        <EmbeddablePanel
+          embeddable={embeddable}
+          getActions={() => Promise.resolve([])}
+          inspector={{ isAvailable: () => false }}
+          hideHeader
+          isRetained
+          isBorderless
+        />
+      </div>
+      <EuiSpacer size="l" />
+      <EuiTitle size="s">
+        <h3>Detector details</h3>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+      {/* {!index && <EuiLoadingSpinner size="l" />} */}
+      {/* Do not initialize until index is available */}
+
+        {/* <EnhancedAccordion
+          id='detectorDetails'
+          title={detectorNameFromVis}
+          // onToggle={detectorDetailsOnToggle}
+          isOpen={isOpen}
+          subTitle={
+            <EuiText size="m">
+              <p>
+                Detector interval: 10 minutes; Window delay: 1 minutesss
+              </p>
+            </EuiText>
+          }>
+          <EuiFormRow label="Detector name">
+            <EuiFieldText value={detectorNameFromVis} />
+          </EuiFormRow>
+        
+          <EuiFormRow label="Detector interval">
+            <EuiSelect
+              id="detectorIntervalSelection"
+              options={intervalOptions}
+              value={intervalValue}
+              onChange={(e) => intervalOnChange(e)}
+            />
+          </EuiFormRow>
+          <EuiFormRow label="Window delay">
+            <EuiSelect
+              id="windowDelaySelection"
+              options={delayOptions}
+              value={delayValue}
+              onChange={(e) => delayOnChange(e)}
+            />
+          </EuiFormRow>
+
+        </EnhancedAccordion> */}
+        <EuiPanel hasBorder={true} hasShadow={true}>
+          <EuiAccordion id='detectorDetails' buttonContent={detectorNameFromVis}>
+            <EuiPanel hasBorder={true} hasShadow={true}>
+              <EuiSpacer size="xs" />
+              <EuiText size="s">
+                Detector interval: 10 minutes; Window delay: 1 minute
```

Comment:
We should pull this from defaults or what is currently set, instead of hardcoding 10/1 here